tensorflow_addons/losses/python/focal_loss.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python.keras import backend as K
```

Comment:
Can you make this import as `import tensorflow as tf`. And then call `tf.keras.backend` when needed. The above import is part of the internal tensorflow API and can be subject to change without notice. We only use the public API interface in Addons.